L320workbook

bdenham opened the pull request:
This PR takes the place of Lab320 to invite last updates and reviews, including Olena T's addition content.

### Reviewer Instructions

Review the following four files. The files will be generated into the PDF lab workbook for Summit:
- src/content/docs/merchants/tutorials/summit-lab-320/index.mdx
- src/content/docs/merchants/tutorials/summit-lab-320/create-product-page.mdx
- src/content/docs/merchants/tutorials/summit-lab-320/create-product-campaign.mdx
- src/content/docs/merchants/tutorials/summit-lab-320/create-experiment.mdx

anthoula committed:
USF-1424: Docs: Multi-website and Multi-storeview implementation with Edge Delivery

- Add Merchant section with multistore
- Add Step by Step

keharper committed:
Re-org and rewrite

keharper committed:
Add GeoIp section

keharper committed:
Add code to sample

keharper committed:
Fix broken link, review comment

keharper committed:
Add comments

keharper committed:
Merge pull request #197 from commerce-docs/fastly

Add GeoIp section

keharper committed:
Review comments

keharper committed:
Apply suggestions from code review

Co-authored-by: Bruce Denham <bdenham@adobe.com>

Commit:
Removed unused prop shippingMethodsSlot, inherited from the old and deleted Checkout container

keharper committed:
Fix code sample

Commit:
Added TermsAndConditions documentation

keharper committed:
Merge branch 'multistore-develop' into multistore

keharper committed:
Merge pull request #193 from commerce-docs/multistore

USF-1424: Docs: Multi-website and Multi-storeview implementation with…

keharper committed:
checkpoint

jeff-matthews committed:
Merge branch 'develop' into multistore-develop

keharper committed:
Merge pull request #205 from commerce-docs/multistore-develop

Merge multistore content into develop

mcocapiza committed:
Update src/content/docs/dropins/checkout/containers/terms-and-conditions.mdx

Co-authored-by: OscarMerino <merino@adobe.com>

mcocapiza committed:
Update src/content/docs/dropins/checkout/containers/terms-and-conditions.mdx

Co-authored-by: OscarMerino <merino@adobe.com>

mcocapiza committed:
Update src/content/docs/dropins/checkout/containers/terms-and-conditions.mdx

Co-authored-by: OscarMerino <merino@adobe.com>

mcocapiza committed:
Update src/content/docs/dropins/checkout/containers/terms-and-conditions.mdx

Co-authored-by: OscarMerino <merino@adobe.com>

mcocapiza committed:
Update src/content/docs/dropins/checkout/containers/terms-and-conditions.mdx

Co-authored-by: OscarMerino <merino@adobe.com>

mcocapiza committed:
Update src/content/docs/dropins/checkout/containers/terms-and-conditions.mdx

Co-authored-by: OscarMerino <merino@adobe.com>

mcocapiza committed:
Update src/content/docs/dropins/checkout/containers/terms-and-conditions.mdx

Co-authored-by: OscarMerino <merino@adobe.com>

Commit:
Merge branch 'develop' of github.com:commerce-docs/microsite-commerce-storefront into USF-2012-New_Terms_and_Conditions_container

bdenham committed:
Merge branch 'USF-2006' into Lab320

bdenham committed:
removed to add it again with a PR

bdenham committed:
add content source for lab320

bdenham committed:
Merge branch 'USF-2006' into Lab320

Commit:
Updated documentation following PR suggestions

bdenham committed:
fix: change extension to .md -> .mdx

bdenham committed:
Merge branch 'Lab320' of github.com:commerce-docs/microsite-commerce-storefront into Lab320

bdenham committed:
fix: move workbook draft to drafts

bdenham committed:
fix: background on docs area selector

keharper committed:
Gated content

bdenham committed:
Apply suggestions from code review

Co-authored-by: Natxo Cabré <natxo.cabre@gmail.com>

bdenham committed:
wip: changes for lesson 1

bdenham committed:
fix: add workshop tips and remove final notes

bdenham committed:
fix: update directory names

bdenham committed:
wip: update lab intro page

Commit:
Splited the documentation in technical and configuration sections

mcocapiza committed:
Update src/content/docs/dropins/checkout/containers/terms-and-conditions.mdx

Co-authored-by: OscarMerino <merino@adobe.com>

Commit:
Minor change

bdenham committed:
fix: update intro page warmup

bdenham committed:
update lab introduction page

bdenham committed:
lesson1: add instructions sans screenshots

bdenham committed:
fix: add content for lessons 2 & 3, updated warmup

keharper committed:
fix link

keharper committed:
Merge branch 'develop' into gated

bdenham committed:
add callout to avoid publishing and update warmup list

keharper committed:
Review feedback

bdenham committed:
update lesson 1 with final pdp

keharper committed:
Apply suggestions from code review

Co-authored-by: Jeff Matthews <matthews.jeffery@gmail.com>

keharper committed:
Review comments

keharper committed:
Merge pull request #208 from commerce-docs/gated

mcocapiza committed:
Update src/content/docs/dropins/checkout/containers/place-order.mdx

Co-authored-by: Daniel Del Carmen <del@adobe.com>

mcocapiza committed:
Update src/content/docs/dropins/checkout/containers/terms-and-conditions.mdx

Co-authored-by: Daniel Del Carmen <del@adobe.com>

mcocapiza committed:
Update src/content/docs/dropins/checkout/containers/terms-and-conditions.mdx

Co-authored-by: Daniel Del Carmen <del@adobe.com>

mcocapiza committed:
Update src/content/docs/dropins/checkout/containers/terms-and-conditions.mdx

Co-authored-by: Daniel Del Carmen <del@adobe.com>

mcocapiza committed:
Update src/content/docs/dropins/checkout/containers/terms-and-conditions.mdx

Co-authored-by: Daniel Del Carmen <del@adobe.com>

mcocapiza committed:
Update src/content/docs/dropins/checkout/containers/terms-and-conditions.mdx

Co-authored-by: Daniel Del Carmen <del@adobe.com>

mcocapiza committed:
Update src/content/docs/releases/index.mdx

Co-authored-by: Daniel Del Carmen <del@adobe.com>

mcocapiza committed:
Update src/content/docs/merchant/terms-and-conditions.mdx

Co-authored-by: Daniel Del Carmen <del@adobe.com>

mcocapiza committed:
Update src/content/docs/merchant/terms-and-conditions.mdx

Co-authored-by: Daniel Del Carmen <del@adobe.com>

mcocapiza committed:
Update src/content/docs/merchant/terms-and-conditions.mdx

Co-authored-by: Daniel Del Carmen <del@adobe.com>

mcocapiza committed:
Update src/content/docs/merchant/terms-and-conditions.mdx

Co-authored-by: Daniel Del Carmen <del@adobe.com>

mcocapiza committed:
Update src/content/docs/releases/index.mdx

Co-authored-by: Daniel Del Carmen <del@adobe.com>

mcocapiza committed:
Update src/content/docs/releases/index.mdx

Co-authored-by: Daniel Del Carmen <del@adobe.com>

mcocapiza committed:
Update src/content/docs/merchant/terms-and-conditions.mdx

Co-authored-by: Daniel Del Carmen <del@adobe.com>

mcocapiza committed:
Update src/content/docs/merchant/terms-and-conditions.mdx

Co-authored-by: Daniel Del Carmen <del@adobe.com>

mcocapiza committed:
Update src/content/docs/merchant/terms-and-conditions.mdx

Co-authored-by: Daniel Del Carmen <del@adobe.com>

mcocapiza committed:
Update src/content/docs/merchant/terms-and-conditions.mdx

Co-authored-by: Daniel Del Carmen <del@adobe.com>

mcocapiza committed:
Update src/content/docs/merchant/terms-and-conditions.mdx

Co-authored-by: Daniel Del Carmen <del@adobe.com>

mcocapiza committed:
Update src/content/docs/dropins/checkout/functions.mdx

Co-authored-by: Daniel Del Carmen <del@adobe.com>

mcocapiza committed:
Update src/content/docs/merchant/terms-and-conditions.mdx

Co-authored-by: Daniel Del Carmen <del@adobe.com>

Commit:
Merge branch 'develop' of github.com:commerce-docs/microsite-commerce-storefront into USF-2012-New_Terms_and_Conditions_container

Commit:
Updated documentation following PR suggestions

Commit:
Updated docs

danidelcar committed:
Merge pull request #204 from mcocapiza/USF-2012-New_Terms_and_Conditions_container

USF-2012: Create Terms and Conditions documentation

bdenham committed:
fix: add lab login section

bdenham committed:
fix: temp update to page title for pdf creation

bdenham committed:
fix: temp update to page title for pdf creation

bdenham committed:
fix: temp update to page title for pdf creation

keharper committed:
Add cart functions and container

bdenham committed:
Lesson 1 complete

bdenham committed:
Apply suggestions from code review

Co-authored-by: Natxo Cabré <natxo.cabre@gmail.com>

bdenham committed:
Update lesson 2 sans screenshots

bdenham committed:
Lesson 3 finished, but not sure about Step 3

keharper committed:
Add gift options/gift card docs

keharper committed:
remove blank line

bdenham committed:
fix: lesson 3 instructions for experimentation

bdenham committed:
add screenshots to lesson 2

bdenham committed:
Update screenshots for lesson 2

bdenham committed:
fix: login credentials for attendees

bdenham committed:
add print css

ericeoeur committed:
updated docs for updateProductsFromCart

bdenham committed:
test: add page breaks for PDF output

bdenham committed:
fix: add pdf page break after each task to avoid bad page breaks

bdenham committed:
feat: add pdf header and footer

bdenham committed:
add details for pdf script generation

bdenham committed:
fix: headers and footers

bdenham committed:
fix: pdf header and footer spacing

bdenham committed:
fix: more spacing fixes for pdf inserted header and footer

bdenham committed:
fix: pdf header and footer spacing again until I solve the localhost pdf generation problem

bdenham committed:
fix: pdf header and footer spacing again until I solve the localhost pdf generation problem

bdenham committed:
fix: pdf header and footer spacing again until I solve the localhost pdf generation problem

bdenham committed:
fix: optimize images

bdenham committed:
fix: tighten steps text

ericeoeur committed:
Update src/content/docs/dropins/cart/functions.mdx

Co-authored-by: Daniel Del Carmen <del@adobe.com>

ericeoeur committed:
Update src/content/docs/dropins/cart/functions.mdx

Co-authored-by: Daniel Del Carmen <del@adobe.com>

bdenham committed:
fix: multiple fixes for pdf output

supernova-at committed:
Create dropin-starter-template.md

Creates the initial draft for using the dropin starter template.

tkacheva commented on _drafts/Lab320Workbook.mdx:
```diff
@@ -0,0 +1,287 @@
+    Congrats on your first featured product page in under 15 minutes!
+
+    > **Screenshot Placeholder:** Final product page preview
+
```
11. **Open a page using in-context editing tool **
    - Open iPhone-13 page in document authoring. 
    - Click on the "open in-context editing" button on the right side of the document content.
12. **Update hero-banner message using in-context editing tool **
    - Double-click on the text message in the hero banner. 
    - Type "Capture breathtaking pictures with new wide lens" in the rich text editor.
    - Click Open Page to preview the change.
13. **Update hero-banner message with GenAI**
    - Select hero-banner message inside of the in-context editor
    - click Generate variations on the right side menu 
    - in the panel that opened on the right, select one of the suggestion "Rewrite content to be more engaging" 
    - Click Generate variations 
    - Choose one of the prompts that you like the most 
    - See the prompt to be applied to the hero-banner messaging
    - Publish the page.
Now you've just taken your content velocity even further with the help of AI!
    - 
    -

bdenham replied:
@tkacheva: I added Steps 10 and 11 to lesson 1 to capture these instructions. See https://commerce-docs.github.io/microsite-commerce-storefront/merchants/tutorials/summit-lab-320/create-product-page/#update-the-hero-banner-with-in-context-editing

bdenham committed:
feat: add steps to lesson 1 for UE and AI editing

jeff-matthews commented on src/content/docs/merchants/tutorials/summit-lab-320/create-product-page.mdx:
```diff
@@ -1,13 +1,226 @@
+### Create the page
+
+<Steps>
+1. Navigate to folder `products`.
```
```suggestion
1. Navigate to the `products` folder.
```

jeff-matthews commented on src/content/docs/merchants/tutorials/summit-lab-320/create-product-page.mdx:
```diff
@@ -1,13 +1,226 @@
+
+<Steps>
+1. Navigate to folder `products`.
+1. Create a folder called `apple-iphone-13`.
```
```suggestion
1. Create a new folder named `apple-iphone-13`.
```

jeff-matthews replied:
```suggestion
1. Create a folder named `apple-iphone-13`.
```

jeff-matthews commented on src/content/docs/merchants/tutorials/summit-lab-320/create-product-page.mdx:
```diff
@@ -1,13 +1,226 @@
+
+<Steps>
+1. Open your `iphone-13` document.
+1. Apply the Edit menu → **Library** → **Templates**.
```
```suggestion
1. From the Edit menu, click **Library** → **Templates**.
```

jeff-matthews commented on src/content/docs/merchants/tutorials/summit-lab-320/create-product-page.mdx:
```diff
@@ -1,13 +1,226 @@
+### Preview the page
+
+<Steps>
+1. Click the instant **Preview** button.
```
I'm assuming that's the paper plane icon based on the lab warm up, but this still made me pause.

bdenham replied:
Actually, no. I'm referring to a different button than the paper plane Action button. So that's a good, valid callout. I need to find a nice solution for inline button images.

jeff-matthews commented on src/content/docs/merchants/tutorials/summit-lab-320/create-product-page.mdx:
```diff
@@ -1,13 +1,226 @@
+<Task>
+### Update the product-details block
+
+The Single Product Detail page template comes with a `product-details` block, also known as a Product Detail Page (PDP) drop-in. Like other document blocks, the PDP drop-in is a two-column table — **property names** on the left, **property values** on the right. Make the following changes to the `product-details` block.
```
```suggestion
The Single Product Detail page template comes with a `product-details` block, also known as a Product Detail Page (PDP) drop-in component. Like other document blocks, the PDP component is a two-column table — **property names** on the left, **property values** on the right. Make the following changes to the `product-details` block.
```

jeff-matthews commented on src/content/docs/merchants/tutorials/summit-lab-320/create-product-campaign.mdx:
```diff
@@ -1,12 +1,156 @@
+
+<Steps>
+1. Insert a new —*Section Break*— directly after the Hero block's `section-metadata` using the **Section Break** tool in the Edit menu.
+1. Insert a **`targeted-block fragment`** from `Library > Blocks > Targeted Blocks`.
```
I don't think Library > Blocks > Targeted Blocks should be code formatted. We use bolding elsewhere.

jeff-matthews commented on src/content/docs/merchants/tutorials/summit-lab-320/create-product-campaign.mdx:
```diff
@@ -1,12 +1,156 @@
+### Preview the Plan-Connect subscribers banner
+
+<Steps>
+1. From the home page preview, select `Account > Login` from the site's navigation menu and enter these credentials:
```
I don't think Account > Login should be code formatted. We use bolding elsewhere.

jeff-matthews commented on src/content/docs/merchants/tutorials/summit-lab-320/create-product-campaign.mdx:
```diff
@@ -1,12 +1,156 @@
+<Steps>
+1. Open the `cart` document in your project root.
+1. Insert a new —*Section Break*— directly after the `commerce-cart` **metadata** block using the **Section Break** tool in the Edit menu.
+1. Insert a **`targeted-block fragment`** from `Library > Blocks > Targeted Blocks`.
```
I don't think Library > Blocks > Targeted Blocks should be code formatted. We use bolding elsewhere.

jeff-matthews commented on src/content/docs/merchants/tutorials/summit-lab-320/create-product-campaign.mdx:
```diff
@@ -1,12 +1,156 @@
+
+<Steps>
+1. Click the **Action** button (paper plane icon), select **Preview**.
+1. Login to the site as a Plan-Connect customer (`Account > Login`) and enter the following credentials:
```
I don't think Account > Login should be code formatted. We use bolding elsewhere.

jeff-matthews reviewed (commented):
Looks great! I like the new nav pattern for developers and merchants.

I just have some minor editorial comments about formatting. The flow makes sense.

The only thing I'm wondering about is: what happens if we deploy this to prod and someone who isn't attending the lab attempts it? 

- Will they be able to do it? If so, for how long before some lab resource is decommissioned? 
- How are you envisioning the transition to evergreen use case docs post-Summit lab?

bdenham commented:
@jeff-matthews, yeah, all great questions. After I generate the PDF for the lab later tonight, I’ll think it through to find a solution for publishing to production tomorrow afternoon. Stay tuned!

bdenham committed:
Update src/content/docs/merchants/tutorials/summit-lab-320/create-product-page.mdx

Co-authored-by: Jeff Matthews <matthews.jeffery@gmail.com>

bdenham committed:
Apply suggestions from code review

Co-authored-by: Jeff Matthews <matthews.jeffery@gmail.com>

bdenham committed:
fix: changes from review and style updates

bdenham committed:
chore: converted and optimized images to webp

bdenham committed:
fix: add images to public for testing

bdenham committed:
fix: update image references to public folder

bdenham committed:
fix: remove diagram widths

bdenham committed:
fix: add maxHeight to Diagram print css

bdenham committed:
fix: add Adobe copyright to footer

bdenham committed:
feat: add all documentation areas to sidebar

bdenham committed:
fix: added maxHeight to lesson 1 final page example

danidelcar committed:
Merge pull request #210 from commerce-docs/USF-2007-Remove-Update-Product

USF-2007 - UpdateProductsFromCart Docs

supernova-at committed:
Update _drafts/dropin-starter-template.md

Co-authored-by: James Calcaben <jcalcaben@users.noreply.github.com>

supernova-at committed:
Update dropin-starter-template.md

Updates instructions to include generating a component, a container, and adding the container to the sandbox env.

supernova-at committed:
Merge branch 'develop' into patch-1

supernova-at committed:
Update _drafts/dropin-starter-template.md

Co-authored-by: Bruce Denham <brucedenham@mac.com>

supernova-at committed:
Update _drafts/dropin-starter-template.md

Co-authored-by: Bruce Denham <brucedenham@mac.com>

supernova-at committed:
Update dropin-starter-template.md

Removes "conclusion" section

keharper committed:
Developer comments

keharper committed:
Merge branch 'develop' into gift-card

keharper committed:
review comments

keharper committed:
Merge branch 'gift-card' of github.com:commerce-docs/microsite-commerce-storefront into gift-card

supernova-at committed:
Update dropin-starter-template.md

Incorporates change suggestions

supernova-at committed:
Apply suggestions from code review

Incorporates suggested fixes

Co-authored-by: Bruce Denham <brucedenham@mac.com>

keharper committed:
heading capitalization

keharper committed:
Merge pull request #209 from commerce-docs/gift-card

Add gift card and gift option docs

bdenham committed:
Merge branch 'develop' into patch-1

bdenham committed:
Merge pull request #212 from supernova-at/patch-1

Create dropin-starter-template.md

bdenham committed:
merged develop with navigation changes and tutorials for lab

bdenham committed:
chore: updated all dependencies, and removed tailwindcss

bdenham committed:
fix: update css for regular bullets

bdenham committed:
fix: update home page labeling

bdenham committed:
fix: add links to videos

bdenham committed:
fix: for summit access

bdenham committed:
fix: nav labels

bdenham committed:
fix: home page links to summit lab

bdenham committed:
fix: add video references and mdx files

bdenham committed:
feat: add creating droping topic

bdenham committed:
fix: pluralize components in title
